editor/src/components/canvas/controls/outline-utils.tsx

```diff
@@ -1,50 +1,10 @@
-import { Size } from '../../../core/shared/math-utils'
-import { UtopiaTheme } from '../../../uuiui'
-import { isFeatureEnabled } from '../../../utils/feature-switches'
+import { CanvasRectangle } from '../../../core/shared/math-utils'
 
-function getDimension(size: Size): 'point' | 'horizontal' | 'vertical' | 'n/a' {
-  if (size.width === 0 && size.height === 0) {
-    return 'point'
-  }
-  if (size.width === 0) {
-    return 'vertical'
-  }
-  if (size.height === 0) {
-    return 'horizontal'
-  }
-  return 'n/a'
-}
-
-export function calculateExtraSizeForZeroSizedElement(
-  size: Size,
-): {
-  extraWidth: number
-  extraHeight: number
-  showingInvisibleElement: boolean
-  borderRadius: number
-  dimension: 'point' | 'horizontal' | 'vertical' | 'n/a'
-} {
-  if (isFeatureEnabled('Invisible Element Controls')) {
-    const extraWidth = size.width === 0 ? UtopiaTheme.invisibleIndicatorSize : 0
-    const extraHeight = size.height === 0 ? UtopiaTheme.invisibleIndicatorSize : 0
-    const showingInvisibleElement = extraWidth !== 0 || extraHeight !== 0
-    const borderRadius = showingInvisibleElement ? UtopiaTheme.invisibleIndicatorSize / 2 : 0
-    const dimension = getDimension(size)
-
-    return {
-      extraWidth,
-      extraHeight,
-      showingInvisibleElement,
-      borderRadius,
-      dimension,
-    }
+export const ZeroControlSize = 5
+export function isZeroSizedElement(frame: CanvasRectangle | null | undefined): boolean {
+  if (frame == null) {
```

Comment:
Is this case even possible? Everywhere that we're calling this is also directly reading values from within the `frame`, so if `frame` can be null we should guard against that (I'm bringing this up because IMO it doesn't make sense to return `false` here)

Reply:
fixed